include/repository/plugin_repository.js

```diff
@@ -0,0 +1,204 @@
+var async = require('async');
+var util    = require('../util.js');
+
+module.exports = function PluginRepositoryModule(pb) {
+
+	var PLUGIN_COLL = 'plugin';
+	var GLOBAL_SITE = pb.SiteService.GLOBAL_SITE;
+	var SITE_FIELD = pb.SiteService.SITE_FIELD;
+
+	var publicAPI = {};
```

Comment:
I'd like to get a little documentation on this file. Additionally, it differs a bit from the way the other files are structured.  I was curious if there was a specific reasoning behind that.